client/src/components/confirmationModal/confirmationModal.js

```diff
@@ -71,35 +60,26 @@ class ConfirmationModal extends React.Component {
 
 ConfirmationModal.propTypes = {
   show: PropTypes.bool.isRequired,
-  confirmTitle: PropTypes.string,
-  confirmHeading: PropTypes.node,
-  confirmIcon: PropTypes.node,
-  confirmBody: PropTypes.node,
+  title: PropTypes.string,
+  heading: PropTypes.node,
+  icon: PropTypes.node,
+  body: PropTypes.node,
   confirmButtonText: PropTypes.string,
   cancelButtonText: PropTypes.string,
   onConfirm: PropTypes.func,
   onCancel: PropTypes.func
 };
+
 ConfirmationModal.defaultProps = {
-  confirmTitle: 'Confirm',
-  confirmHeading: null,
-  confirmBody: null,
-  confirmIcon: <Icon type="pf" name="warning-triangle-o" />,
+  title: 'Confirm',
+  heading: null,
+  body: null,
+  icon: <Icon type="pf" name="warning-triangle-o" />,
   confirmButtonText: 'Confirm'
 };
 
-function mapStateToProps(state, ownProps) {
-  return {
-    show: state.confirmationModal.show,
-    confirmTitle: state.confirmationModal.title,
-    confirmHeading: state.confirmationModal.heading,
-    confirmIcon: state.confirmationModal.icon,
-    confirmBody: state.confirmationModal.body,
-    confirmButtonText: state.confirmationModal.confirmButtonText,
-    cancelButtonText: state.confirmationModal.cancelButtonText,
-    onConfirm: state.confirmationModal.onConfirm,
-    onCancel: state.confirmationModal.onCancel
-  };
-}
+const mapStateToProps = function(state) {
+  return { ...state.confirmationModal };
+};
```

Comment:
Acknowledging the naming was probably done to avoid conflicts with html elements... since the overlap wasn't within this component went ahead and converted it